aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -290,6 +290,14 @@ impl VMChangeSet {
             .extend(additional_aggregator_writes)
     }
 
+    // Called by `into_transaction_output_with_materialized_writes` only.
+    pub(crate) fn extend_resource_write_set(
+        &mut self,
+        group_writes: impl Iterator<Item = (StateKey, WriteOp)>,
```

Comment:
Shouldn't group writes be an iterator over (StateKey, StructTag, WriteOp)?

Reply:
These are materialized writes, so groups are combined. Maybe change the argument name?

Reply:
called it finalized_group_writes